ci/recipe/meta.yaml

```diff
@@ -24,6 +24,7 @@ requirements:
     - pandas
     - tzlocal
     - python-dateutil
+    - bibtexparser
```

Comment:
Looks like this is up on (at least) [conda-forge](https://anaconda.org/conda-forge/bibtexparser), so we should be good with adding this new dep, eh?

Reply:
that's the idea